packages/astro/test/errors.test.js

```diff
@@ -21,231 +21,153 @@ before(async () => {
 
 describe('Error display', () => {
   describe('Astro', () => {
-    // This test is redundant w/ runtime error since it no longer produces an Astro syntax error
-    it.skip('syntax error', async () => {
-      if (isMacOS) return;
-
+    it('syntax error in template', async () => {
+      // if (isMacOS) return;
       const res = await fixture.fetch('/astro-syntax-error');
-
-      // 500 returned
       expect(res.status).to.equal(500);
+      const body = await res.text();
+      console.log(res.body);
+      expect(body).to.include('Unexpected &quot;}&quot;');
+    });
 
-      // error message includes "unrecoverable error"
+    it('syntax error in frontmatter', async () => {
+      // if (isMacOS) return;
+      const res = await fixture.fetch('/astro-frontmatter-syntax-error');
+      expect(res.status).to.equal(500);
       const body = await res.text();
       console.log(res.body);
-      expect(body).to.include('unrecoverable error');
+      expect(body).to.include('Unexpected end of frontmatter');
     });
 
     it('runtime error', async () => {
-      if (isMacOS) return;
-
+      // if (isMacOS) return;
```

Comment:
note: i can run these locally, so I figured I'd uncomment to see what happens. 

If they pass, I'll remove entirely.
If they fail, I'll uncomment.

Reply:
update, tests are passing in CI so I'll uncomment